src/components/modules/saver.ts

```diff
@@ -104,10 +104,16 @@ export default class Saver extends Module {
         return;
       }
 
-      blocks.push({
+      const output: any = {
         type: tool,
         data,
-      });
+      };
+
+      if (!_.isEmpty(tunes)) {
+        output.tunes = tunes;
```

Comment:
maybe we need to preserve the `tunes` field event it is empty?

Reply:
I think better to omit it to decrease output's size and explicitly state that block doesn't have any stateful tunes